### ISSUE TITLE
Playback delay

### ISSUE THREAD

**Issue by bitbrain** [utopia-rise/fmod-gdnative#91]
There seems to be a slight delay when playing sounds for me. I did some googling and a possible problem might be the buffer size configured ([see Unity forum](https://qa.fmod.com/t/set-buffer-size/13465))

- what is the current default buffer size configured when using this plugin?
- what would be the best way to change the buffer size / make it configurable?

**Comment by bitbrain** [comment]
I was unable to build the PRs I raised due to the current Github actions setup. In the meantime, I have been granted contributor access to create branches on this repository until https://github.com/utopia-rise/fmod-gdnative/issues/95 is closed.

**Comment by CedNaru** [comment]
From the FMOD documentation:

```
The mixer engine block size. Use this to adjust mixer update granularity. 
See below for more information on buffer length vs latency.

Units: Samples Default: 1024
```


```
bufferlength
Address of a variable that receives the mixer engine block size in samples. 
Default = 1024. (milliseconds = 1024 at 48khz = 1024 / 48000 * 1000 = 10.66ms). 
This means the mixer updates every 21.3ms. Optional. Specify 0 or NULL to ignore.
```

**Comment by bitbrain** [comment]
Yes, after setting the DSP buffer size manually by using the method provided by my PR it fixed the delay. Those 50ms are most likely theoretical and depend on the system and drivers, too.
